posthog/models/cohort/util.py

```diff
@@ -370,6 +372,7 @@ def get_cohort_size(cohort: Cohort, override_version: Optional[int] = None) -> O
             "version": override_version if override_version is not None else cohort.version,
             "team_id": cohort.team_id,
         },
+        workload=Workload.OFFLINE,
```

Comment:
Seems like this was previously being run once on the offline cluster here https://github.com/PostHog/posthog/blob/d64e2cf074e82fcd3a2d65b7fab90edbd7d3791c/posthog/models/cohort/util.py#L302 and then back to the online/default cluster here https://github.com/PostHog/posthog/blob/d64e2cf074e82fcd3a2d65b7fab90edbd7d3791c/posthog/models/cohort/util.py#L331 due to the change to the `kind` tag above.